### PR TITLE
CallbackReturn Fix

### PR DESCRIPTION
**Pull request by kutlayacar**
I have fixed "CallbackReturn" as it mentioned in this [Issue](https://github.com/ros-controls/ros2_control_demos/issues/171).

### CODE REVIEW

**Comment by homalozoa** [comment]
Hi @kutlayacar . Thanks for you contribution. It seems you have added `.idea` folder to this change, please remove it and push again. Thank you.

**Review by homalozoa — CHANGES_REQUESTED** [review]
Remove `.idea`, please.

**Review by harderthan — APPROVED** [review]

**Comment by kutlayacar** [comment]
Hi @homalozoa. Sorry for my mistake. I have deleted .idea folder and pushed again. Thank you.

**Review by homalozoa — APPROVED** [review]
LGTM. Thanks @kutlayacar

**Review by arne48 — APPROVED** [review]
With the .idea clean-up this should be a good fix for #171 
Thank you very much :)

**Comment by kutlayacar** [comment]
Thank you!

**Review by destogl — CHANGES_REQUESTED** [review]
Thanks for the contribution!

You should also update `CallbackReturn` to `hardware_interface::CallbackReturn` in the hardware.

**Comment by kutlayacar** [comment]
You're welcome! 

I guess it should work now. I have forgotten to fix the `CallbackReturn` to `hardware_interface::CallbackReturn` in the .hpp files.

File: ros2_control_demo_hardware/include/ros2_control_demo_hardware/diffbot_system.hpp [hunk 1]
```diff
@@ -33,15 +33,14 @@
 
 namespace ros2_control_demo_hardware
 {
-using CallbackReturn = rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn;
```
Comment (destogl) [inline ros2_control_demo_hardware/include/ros2_control_demo_hardware/diffbot_system.hpp:36]:
```suggestion
```

File: ros2_control_demo_hardware/include/ros2_control_demo_hardware/external_rrbot_force_torque_sensor.hpp [hunk 2]
```diff
@@ -32,24 +32,23 @@
 
 namespace ros2_control_demo_hardware
 {
-using CallbackReturn = rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn;
```
Comment (destogl) [inline ros2_control_demo_hardware/include/ros2_control_demo_hardware/external_rrbot_force_torque_sensor.hpp:35]:
```suggestion
```

File: ros2_control_demo_hardware/include/ros2_control_demo_hardware/rrbot_actuator.hpp [hunk 3]
```diff
@@ -33,15 +33,14 @@
 
 namespace ros2_control_demo_hardware
 {
-using CallbackReturn = rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn;
```
Comment (destogl) [inline ros2_control_demo_hardware/include/ros2_control_demo_hardware/rrbot_actuator.hpp:36]:
```suggestion
```

File: ros2_control_demo_hardware/include/ros2_control_demo_hardware/rrbot_system_multi_interface.hpp [hunk 4]
```diff
@@ -35,15 +35,14 @@
 
 namespace ros2_control_demo_hardware
 {
-using CallbackReturn = rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn;
```
Comment (destogl) [inline ros2_control_demo_hardware/include/ros2_control_demo_hardware/rrbot_system_multi_interface.hpp:38]:
```suggestion
```

File: ros2_control_demo_hardware/include/ros2_control_demo_hardware/rrbot_system_position_only.hpp [hunk 5]
```diff
@@ -30,18 +30,17 @@
 
 namespace ros2_control_demo_hardware
 {
-using CallbackReturn = rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn;
```
Comment (destogl) [inline ros2_control_demo_hardware/include/ros2_control_demo_hardware/rrbot_system_position_only.hpp:33]:
```suggestion
```

File: ros2_control_demo_hardware/include/ros2_control_demo_hardware/rrbot_system_with_sensor.hpp [hunk 6]
```diff
@@ -33,18 +33,17 @@
 
 namespace ros2_control_demo_hardware
 {
-using CallbackReturn = rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn;
```
Comment (destogl) [inline ros2_control_demo_hardware/include/ros2_control_demo_hardware/rrbot_system_with_sensor.hpp:36]:
```suggestion
```

**Review by destogl — APPROVED** [review]
Thanks!!!

**Comment by destogl** [comment]
@kutlayacar can you please fix formatting? (you can use "pre-commit" command in your local repository to do this automatically. (check the internet for starting manuals)

**Comment by kutlayacar** [comment]
> @destogl I am on it!
I have used your .clang-format file to format the code and pushed it. I hope it works like this. Otherwise I need from you a detailed information. Thank you and I am sorry for these issues.

**Comment by kutlayacar** [comment]
> > @destogl I am on it!
> 
> I have used your .clang-format file to format the code and pushed it. I hope it works like this. Otherwise I need from you a detailed information. Thank you and I am sorry for these issues.

So it didn't work properly. I'll commit it again as soon as possible, when I fix it.

**Comment by kutlayacar** [comment]
Hello again, 

the code passed the clang-format test on my local repository. I have committed the changes and hope it works now with no issues. 

Best wishes

**Pull request merged**